File: leetcode/arrayrotation.py
from typing import List
def solution(A: List, K):
    # Implement your solution here
    for i in range(K):
        lastVal = A.pop()
        A.insert(0, lastVal)
    return A


input = [3, 8, 9, 7, 6]
K = 3
print(solution(input, K))

File: codility/arrayRotation.ts
function arrayRotation(A: number[], K: number): number[] {
    // Implement your solution here
    if (K > A.length) {
        K = K % A.length;
    }
    const sliceIndex = A.length - K;
    const firstSlice = A.slice(0, sliceIndex);
    const secondSlice = A.slice(sliceIndex);
    return [...secondSlice, ...firstSlice];
}
